week3/70_climb_stairs.py

```diff
@@ -0,0 +1,12 @@
+# https://leetcode.com/problems/climbing-stairs/description/
+def climb_stairs(self, n: int) -> int:
+    def recursive(n, memoize):
```

Comment:
Simple and nice. Please consider a non-recursive version when you see these kinds of problem next time.